.github/workflows/kubernetes.yml

```diff
@@ -48,33 +52,33 @@ jobs:
           curl -sL -o /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.24.1/bin/linux/amd64/kubectl
           chmod +x /tmp/kubectl
           mv /tmp/kubectl /usr/local/bin/
-      - name: Create cluster
+      - name: Build and Redeploy
         run: |
-          kind create cluster
-      - name: Build and Load Docker images
+          cd kubernetes/linera-validator
+          ./build_and_redeploy.sh
+      - name: Port Forward
         run: |
-          cd docker/local-tests
-          cp ../../target/release/{linera,linera-proxy,linera-server} .
-          for image in linera linera-proxy linera-server setup; do
-              docker build -t "linera-test-$image" --target "$image" .
-              kind load docker-image "linera-test-$image"
-          done
-      - name: Deploy cluster
+          kubectl get pods | grep validator | awk '{ print $1 }' | xargs -J % kubectl port-forward % 19100:19100 &
+      - name: Send sync-balance request
         run: |
-          cd docker/local-tests
-          ./integration-test.sh 4 5
-          kubectl apply -f linera-k8s.yml
-      - name: Wait for client test
-        run: |
-          kubectl wait --for condition=Complete --timeout 180s job/client
+          cd target/release
+          ./linera sync-balance
```

Comment:
You probably need `--wallet` here to point to `/tmp/wallet.json`. Same for `--storage`.

Reply:
Yeah, I'm leaving this as a Draft because I'm using it to test ha
Is there an easy way to run it locally? There probably is, but I didn't research for it